en/tutorials/tutorial_hello_sky.md

```diff
@@ -199,70 +256,70 @@ The `sensor_sub_fd` is a topic handle and can be used to very efficiently perfor
+    { .fd = sensor_sub_fd,   .events = POLLIN },
 };
 
 while (true) {
-	/* wait for sensor update of 1 file descriptor for 1000 ms (1 second) */
-	int poll_ret = px4_poll(fds, 1, 1000);
+uORB/* wait for sensor update of 1 file descriptor for 1000 ms (1 second) */
+uORBint poll_ret = px4_poll(fds, 1, 1000);
 ..
-	if (fds[0].revents & POLLIN) {
-		/* obtained data for the first file descriptor */
-		struct sensor_combined_s raw;
-		/* copy sensors raw data into local buffer */
-		orb_copy(ORB_ID(sensor_combined), sensor_sub_fd, &raw);
-		PX4_INFO("Accelerometer:\t%8.4f\t%8.4f\t%8.4f",
-					(double)raw.accelerometer_m_s2[0],
-					(double)raw.accelerometer_m_s2[1],
-					(double)raw.accelerometer_m_s2[2]);
-	}
+    if (fds[0].revents & POLLIN) {
+        /* obtained data for the first file descriptor */
+        struct sensor_combined_s raw;
+        /* copy sensors raw data into local buffer */
+        orb_copy(ORB_ID(sensor_combined), sensor_sub_fd, &raw);
+        PX4_INFO("Accelerometer:\t%8.4f\t%8.4f\t%8.4f",
+                    (double)raw.accelerometer_m_s2[0],
+                    (double)raw.accelerometer_m_s2[1],
+                    (double)raw.accelerometer_m_s2[2]);
+    }
 }
 ```
 
-Compile the app now by issuing
+Compile the app again by entering:
 
-<div class="host-code"></div>
-
-```
-  make
+```sh
+make
 ```
 
 ### Testing the uORB Subscription
 
-The final step is to start your application as background application.
+The final step is to start your application as a background process/task.
 
-```
-  px4_simple_app &
+```sh
+px4_simple_app &
 ```
 
-Your app will flood the console with the current sensor values:
+Your app will display 5 sensor values in the console and then exit:
 
+```sh
+[px4_simple_app] Accelerometer:   0.0483          0.0821          0.0332
+[px4_simple_app] Accelerometer:   0.0486          0.0820          0.0336
+[px4_simple_app] Accelerometer:   0.0487          0.0819          0.0327
+[px4_simple_app] Accelerometer:   0.0482          0.0818          0.0323
+[px4_simple_app] Accelerometer:   0.0482          0.0827          0.0331
+[px4_simple_app] Accelerometer:   0.0489          0.0804          0.0328
 ```
-  [px4_simple_app] Accelerometer:   0.0483          0.0821          0.0332
-  [px4_simple_app] Accelerometer:   0.0486          0.0820          0.0336
-  [px4_simple_app] Accelerometer:   0.0487          0.0819          0.0327
-  [px4_simple_app] Accelerometer:   0.0482          0.0818          0.0323
-  [px4_simple_app] Accelerometer:   0.0482          0.0827          0.0331
-  [px4_simple_app] Accelerometer:   0.0489          0.0804          0.0328
-```
 
-It will exit after printing five values. The next tutorial page will explain how to write a deamon which can be controlled from the commandline.
+> **Tip** The **Firmware/src/examples/px4_daemon_app** example shows how to write a daemon (background process) that can be controlled from the commandline.
```

Comment:
Note, the daemon doc no longer exists, but still useful to point to the example here.